components/statustty/pkg/systemd.go

```diff
@@ -35,7 +35,7 @@ type Systemd struct {
 }
 
 func NewSystemd() (*Systemd, error) {
-	conn, err := dbus.NewSystemdConnection()
```

Comment:
What's the reasoning behind this change? Originally I used a direct connection to enable statustty to run as early in the boot process as possible.